lib/hardware_services/network_drivers/HilNetworkDriver.py

```diff
@@ -14,19 +14,74 @@
 from subprocess import call
 from subprocess import check_call
 
-from hardware_services.network_service import NetworkService
+sys.path.append(os.path.join(os.path.dirname(__file__), "..", "util"))
 
-# added for EC528 HIL-QUADS integration project
-hil_url = 'http://127.0.0.1:5000'
+import hilapi
 
 class HilNetworkDriver(NetworkService):
 
+    def __read_current_state(self, argv):
+        """
+        Reads actual current network of a node from the HIL database and returns name of this network
+        """
+        
+        pass
+
+    def __write_current_state(self, argv):
+        """
+        Writes/updates information about a new network of a node to the HIL database
+        """
+        
+        pass
+        
+    def __move_one_host(self, argv):
+        if len(argv) != 3:
+            sys.exit("Incorrect number of arguments. Should be host_to_move old_cloud new_cloud.")
```

Comment:
Call `sys.exit` in a library makes it difficult to know how to use the library in different situations.  Instead, raise and exception.

However, if `__move_one_host` is only called within this module, why not just explicitly list all the arguments?   Python has variable length argument support built in, so using a list is not considered "pythonic".